server/app/models/stack.rb

```diff
@@ -0,0 +1,26 @@
+class Stack
+  include Mongoid::Document
+  include Mongoid::Timestamps
+  include Mongoid::Enum
+
+  field :name, type: String
+  field :version, type: String
+  enum :state, [:initialized, :deployed, :terminated]
+
+  belongs_to :grid
+
+  has_many :grid_services
+  #has_many :audit_logs
```

Comment:
Let's remove these commented code blocks... if we don't need them then they should not be here :)